examples/hello_world.js

```diff
@@ -69,14 +69,13 @@ function main() {
   }
 
   // Delete objects we created through |new|. We just do a few of them here, but you should do them all if you are not shutting down ammo.js
-  Ammo.destroy(collisionConfiguration);
-  Ammo.destroy(dispatcher);
-  Ammo.destroy(overlappingPairCache);
+  Ammo.destroy(dynamicsWorld);
```

Comment:
why? in general the order of mallocs does not need to be the same as free. except here, perhaps the destructor of the main object assumes its children still exist. so there is a reason here, I think, but it's specific to these objects, not to malloc/free in general. so a comment might help people see that.